tests/capi/unittest_capi_service_agent_client.cc

```diff
@@ -1156,6 +1171,69 @@ TEST_F (MLServiceAgentTest, model_scenario)
   g_free (test_model2);
 }
 
+/**
+ * @brief Negative testcase of pipeline gdbus call.
+ */
+TEST_F (MLServiceAgentTest, pipeline_gdbus_call_n)
+{
+  int ret;
+  GError *error = NULL;
+
+  MachinelearningServicePipeline *proxy_for_pipeline = machinelearning_service_pipeline_proxy_new_for_bus_sync (
+    G_BUS_TYPE_SESSION, G_DBUS_PROXY_FLAGS_NONE,
+    "org.tizen.machinelearning.service",
+    "/Org/Tizen/MachineLearning/Service/Pipeline", NULL, &error);
+
+  if (!proxy_for_pipeline || error) {
+    g_critical ("Failed to create proxy_for_pipeline for machinelearning service pipeline");
+    if (error) {
+      g_critical ("Error Message : %s", error->message);
+      g_clear_error (&error);
+    }
+    EXPECT_TRUE (false);
```

Comment:
How about using ASSERT_TRUE (proxy_for_pipeline != NULL);

Reply:
Those lines are changed to ASSERT_TRUE, Thanks